client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -564,7 +565,11 @@ export const FeatureRow = React.memo(
     return (
       <>
         <tr>
-          <th className="bc-feature" scope="row">
+          <th
+            className="bc-feature"
+            style={{ borderLeftWidth: borderleftwidth }}
```

Comment:
... avoid inline styles:

```suggestion
            className={bc-feature bc-feature-depth-${depth}}
```

And then add style for `.bc-feature-depth-3` and possibly `.bc-feature-depth-2` to the `index.scss` in the same folder.